full-stack-tests/backend/src/integration/CloudSqlite.test.ts

```diff
@@ -235,6 +235,22 @@ describe("CloudSqlite", () => {
     testContainer2.disconnect();
   });
 
+  it("Should refresh write lock for the current user when uploading changes", async () => {
+    const testContainer1 = testContainers[3];
+    const testCache1 = azSqlite.makeCaches(["testCache1"])[0];
+    testContainer1.connect(testCache1);
+    testContainer1.acquireWriteLock(user1);
+    await testContainer1.copyDatabase("testBim", "testBimCopy1");
+    const db1 = await BriefcaseDb.open({ fileName: "testBimCopy1", container: testContainer1 });
+    db1.saveFileProperty({ name: "refresh", namespace: "refreshTest", id: 1, subId: 1 }, "this is a test");
+    db1.close();
+    await azSqlite.setWriteLockToExpired(testContainer1, "bcv_kv.bcv");
+    // there are no other users, uploadChanges should refresh the write lock for the current user.
```

Comment:
What about when writeLock isn't expired?

Reply:
added in [commit 0bd6507cf820f33d8598b933ffe4f75734b80465](https://github.com/iTwin/itwinjs-core/pull/7000/commits/0bd6507cf820f33d8598b933ffe4f75734b80465)